README.md

```diff
@@ -457,8 +457,22 @@ The performance tests can similarly be enabled. To execute them and see their ou
 $ ctest --verbose -L performance
 ```
 
+
 ### Configuring
-The 'conf' directory in the installation root contains a template config.yml document, minifi.properties, and minifi-log.properties. Please see our [Configuration document](CONFIGURE.md) for details on how to configure agents.
+The 'conf' directory in the installation root contains all configuration files.
+
+The files conf/minifi.properties, conf/minifi-log.properties and conf/minifi-uid.properties contain key-value pair configuration settings;
+these are the default settings supplied by the latest MiNiFi version. If you would like to modify these, you should create a corresponding
+.d directory (e.g. conf/minifi.properties.d) and put your settings in a new file inside this directory. These files are read and applied
+in lexicographic order, after the default settings file.
+The Windows installer creates a conf/minifi.properties.d/10_installer_properties file, which contains C2 connection settings.
```

Comment:
The files under minifi.properties.d should have the .properties extension too by convention, but it's OK if the minifi service doesn't enforce this and reads all files under the directory.
```suggestion
The Windows installer creates a conf/minifi.properties.d/10_installer.properties file, which contains C2 connection settings.
```

Reply:
I don't know about convention: most of the files under `/etc/*.d` on my machine have no extension, with `.conf` a distant second and many others like `.driver`, `.sources` etc. I have changed them to `.properties` anyway, in 12e6d407c245e03bbbe8f627cbc85df9abe5495f.

When reading them, we will continue to read all files in the `.d` directory except backup files.